constants.py

```diff
@@ -29,9 +29,12 @@
 NS_POLICY_ANNOTATION = "net.beta.kubernetes.io/network-policy"
 
 # Tier name /order to use for policies.
-NET_POL_TIER_NAME = "k8s-network-policy"
+NET_POL_TIER_DEFAULT = "default"
```

Comment:
As we spoke, I think it doesn't buy us much, I'll pull the value out of constants and remove them.